<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zoi_type_dialog</class>
 <widget class="QDialog" name="zoi_type_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZOI Type</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../icon.png</normaloff>../../icon.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ZOI Type Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_zoi_type_name">
     <property name="placeholderText">
      <string>E.g.,  Bank Erosion</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ZOI Type Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit_zoi_type_description">
     <property name="placeholderText">
      <string>Description of hydrologic,  hydraulic,  geomorphic,  biologica,l  and/or ecological processes.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
